core/chains/evm/config/toml/defaults/Gnosis_Mainnet.toml

```diff
@@ -13,7 +13,3 @@ LogPollInterval = '5s'
 [GasEstimator]
 PriceDefault = '1 gwei'
 PriceMax = '500 gwei'
-# 1 Gwei is the minimum accepted by the validators (unless whitelisted)
```

Comment:
I think this comment is valuable and we might want to keep it regardless of the config

Reply:
Ah I misread it as `wei` :facepalm:

Reply:
Fallback is 1 gwei so it can be removed, I was just referring to the comment.